services/external_sources/who.py

```diff
@@ -50,15 +44,17 @@ def who_search(query, owner, limit=5, prev=0):
     return res
 
 
-def _who_search_indicators(query):
+def _who_search_indicators(query, skip, top):
```

Comment:
nice.
Since the other functions have a "what if there is no search term" clause, perhaps good to add here that it is not necessary as it returns data by default?